### PR TITLE
Description can be mapped to a custom field

### PR DESCRIPTION
**Pull request by zztimur**
Mapping for description field. Set to 'description' product attribute by default.

@skwp Should the description update? or should it have a setting for update:yes/no, same as price,title?

### CODE REVIEW

File: app/code/community/Reverb/ReverbSync/Model/Mapper/Product.php [hunk 1]
```diff
@@ -106,6 +106,14 @@ public function getProductPrice($product)
         return $product->getPrice();
     }
 
+    public function getProductDescription($product)
+    {
+        $_reverb_allowed_tags = '<ul>,<ol>,<li>,<strong>,<b>,<em>,<i>,<u>,<div>,<p>,<br>';
+        $_descr = Mage::helper('core')->stripTags($this->getProductValueForListing($product, 'description'), $_reverb_allowed_tags);
```
Comment (skwp) [inline app/code/community/Reverb/ReverbSync/Model/Mapper/Product.php]: there should be a fallback here to the normal description field?

**Comment by skwp** [comment]
I think we should have a yes/no setting for update, defaulted to no

**Comment by skwp** [comment]
thanks for the help here

**Comment by zztimur** [comment]
I haven't included this to be an updated attribute.

File: app/code/community/Reverb/ReverbSync/Model/Mapper/Product.php (hunk 1 above)
Comment (zztimur) [inline app/code/community/Reverb/ReverbSync/Model/Mapper/Product.php]: Please be more specific.

Comment (skwp) [inline app/code/community/Reverb/ReverbSync/Model/Mapper/Product.php]: nevermind, as discussed in chat - you are setting the mapping automatically as a default.

curious why you're not using the getMagentoProductAttributeForReverbField function here?

Comment (zztimur) [inline app/code/community/Reverb/ReverbSync/Model/Mapper/Product.php]: Currently that function only works for the mapped items:
- Make
- Model
- Price
- Description.

Comment (zztimur) [inline app/code/community/Reverb/ReverbSync/Model/Mapper/Product.php]: Oh, okay, that's a different function. Anyway, the one you're referring returns the product attribute - mapped field.

**Pull request merged**